outlines/react.md

```diff
@@ -0,0 +1,34 @@
+# React
+
+1. Introduction to React
+  1. Background: developed by Facebook, used widely in production, vibrant ecosystem
+  2. Installing and using React in Meteor 1.3
+  3. A simple example
+
+2. JSX in depth
+  1. What JSX really is (helping people understand it)
+  2. JSX workflow (transpiling - we've got it covered for you!)
+  3. JavaScript expressions inside JSX
+  4. Conditional and list rendering with JSX
+  5. Gotchas
+
+3. Components in depth
+  1. State
+  2. Props
+  3. Lifecycle
+  4. Passing callbacks
+  5. Stateless components
```

Comment:
Can you flesh this out a bit more @yyx990803 - we have written a lot in the [UI guide](http://guide.meteor.com/ui-ux.html) about smart vs reusuable vs pure components (which in the React world has different terminology appearing now, maybe container vs X vs presentational -- I'm not sure if you are "allowed" to use state any more). 

It'll be interesting to see how the recommendations align -- and if they don't should we discuss changing the UI guide, which is intended to be the higher-order concepts that are universal across rendering frameworks.